streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImpl.java

```diff
@@ -29,6 +29,7 @@
 import org.apache.kafka.streams.state.Stores;
 
 import java.util.Collections;
+import java.util.Objects;
```

Comment:
Just a suggestion: use ``import static java.util.Objects.requireNonNull`` so that you can save some keystrokes (and reduce clutter) down by using:

```
     requireNonNull(reducer, "reducer can't be null");
```
instead of 
```
Objects.requireNonNull(reducer, "reducer can't be null");
``` 

But this is just a stylistic recommendation, up to you. :)